unit_tests/input.nml

```diff
@@ -0,0 +1,89 @@
+&amip_interp_nml
+       interp_oi_sst = .true.
+       use_ncep_sst = .true.
+       use_ncep_ice = .false.
+       no_anom_sst = .false.
+       data_set = 'reynolds_oi',
+       date_out_of_range = 'climo',
+/
+
+&atmos_model_nml
+       blocksize = 32
+       chksum_debug = .false.
+       dycore_only = .false.
+       fhout = 1
+       fhmax = 800
+       fhmaxhf = -1
+       ccpp_suite = 'FV3_GFS_v15p2_coupled'
+/
+
+&diag_manager_nml
+       prepend_date = .F.
+/
+
+&fms_io_nml
+       checksum_required   = .false.
+       max_files_r = 100,
+       max_files_w = 100,
+/
+
+&fms_nml
+       clock_grain = 'ROUTINE',
+       domains_stack_size = 40000000,
+       print_memory_usage = .false.
+/
+
+&fv_grid_nml
+       grid_file = 'INPUT/grid_spec.nc'
+/
+
+&fv_core_nml
+       layout   = 1,4
+       io_layout = 1,1
+       npx      = 97,
+       npy      = 97,
+       ntiles   = 6,
+       grid_type = -1
+
+/
+
+&gfs_physics_nml
+       do_sppt        = .true.
+       do_shum        = .true.
+       do_skeb        = .true.
+       lndp_type      = 2
+       n_var_lndp     = 2
+/
+
+&nam_sppperts
```

Comment:
Here are the &nam_sppperts options for SPP (not that these are the generally recommended settings to run SPP at 3-km in the SRW App; I've not tested it with any other UFS App or at different grid resolutions):

  DO_SPP: true
  SPP_VAR_LIST: [ "pbl", "sfc", "mp", "rad", "gwd" ]
  SPP_MAG_LIST: [ 0.2, 0.2, 0.75, 0.2, 0.2 ] #Variable "spp_prt_list" in input.nml
  SPP_LSCALE: [ 150000.0, 150000.0, 150000.0, 150000.0, 150000.0 ]
  SPP_TSCALE: [ 21600.0, 21600.0, 21600.0, 21600.0, 21600.0 ]
  SPP_SIGTOP1: [ 0.1, 0.1, 0.1, 0.1, 0.1 ]
  SPP_SIGTOP2: [ 0.025, 0.025, 0.025, 0.025, 0.025 ]
  SPP_STDDEV_CUTOFF: [ 1.5, 1.5, 2.5, 1.5, 1.5 ]
  ISEED_SPP: [ 4, 5, 6, 7, 8 ]

Note that EMC implemented SPP for the Grell-Frietas scheme, which can be turned on by adding "cu_deep" to the SPP_VAR_LIST variable and adding an additional value to the other namelist entries.

Reply:
Ok, @JeffBeck-NOAA. I've added the changes to input.nml. I added "cu_deep" as an additional variable to SPP_VAR_LIST as well.

Reply:
@zach1221, did n_var_spp not work with 6?

Reply:
@JeffBeck-NOAA yes, looks like it failed with the below. I was going to try with 5, but Hera is down for maintenance currently.

At line 182 of file compns_stochy.F90 (unit = 21, file = 'input.nml')
 1: Fortran runtime error: Cannot match namelist object name spp_var_list: